src/expr.rs

```diff
@@ -35,7 +37,222 @@ pub enum Expr {
     Geometry(Geometry),
 }
 
+impl From<Value> for Expr {
+    fn from(v: Value) -> Expr {
+        let e: Expr = serde_json::from_value(v).unwrap();
+        e
+    }
+}
+
+impl From<Expr> for Value {
+    fn from(v: Expr) -> Value {
+        let v: Value = serde_json::to_value(v).unwrap();
+        v
+    }
+}
+
+
+impl TryInto<f64> for Expr {
+    type Error = ();
+    fn try_into(self) -> Result<f64, Self::Error> {
+        match self {
+            Expr::Float(v) => Ok(v),
+            Expr::Literal(v) => f64::from_str(&v).or(Err(())),
+            _ => Err(()),
+        }
+    }
+}
+
+impl TryInto<bool> for Expr {
+    type Error = ();
+    fn try_into(self) -> Result<bool, Self::Error> {
+        match self {
+            Expr::Bool(v) => Ok(v),
+            _ => Err(()),
+        }
+    }
+}
+
 impl Expr {
+    /// Insert values from properties from json
+    ///
+    ///  # Examples
+    ///
+    /// ```
+    /// use serde_json::{json, Value};
+    /// use cql2::Expr;
+    /// let item = json!({"properties":{"eo:cloud_cover":10, "datetime": "2020-01-01 00:00:00Z", "boolfield": true}});
+    /// let mut expr_json = json!(
+    ///     {
+    ///         "op": "+",
+    ///         "args": [
+    ///             {"property": "eo:cloud_cover"},
+    ///             10
+    ///         ]
+    ///     }
+    /// );
+    ///
+    /// let mut expr: Expr = serde_json::from_value(expr_json).unwrap();
+    /// println!("Initial {:?}", expr);
```

Comment:
I know we're WIP testing, but I find that `println` debugging is quicker+better when it's in library tests, not in doctests.

Reply:
Heh. I am well known for copious use of print statements when in WIP mode. Will definitely clear when marking for merge.